src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/AmendReferralService.kt

```diff
@@ -57,4 +59,28 @@ class AmendReferralService(
     return referralService.getSentReferralForUser(id, user)
       ?: throw ResponseStatusException(HttpStatus.NOT_FOUND, "sent referral not found [id=$id]")
   }
+
+  fun getListOfChangeLogEntries(referral: Referral): List<ChangelogValuesDTO> {
+    val changeLogEntities = changelogRepository.findByReferralIdOrderByChangedAtDesc(referral.id)
+    val listOfChangelogValuesDTO: MutableList<ChangelogValuesDTO> = mutableListOf()
```

Comment:
The general pattern we follow is sending back entity objects back to controller and use DTO mapper class to create the DTO that will sent back to client. Could we please do it that way